### PR TITLE
Support PHP7.1+ for Guzzle5 : Rename package

### PR DESCRIPTION
**Pull request by Progi1984**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | Relative to Prestashop/Prestashop#13761
| License       | Apache License 2.0

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "csa/guzzle-bundle",
+    "name": "prestashop/guzzle-bundle",
```
Comment (matks) [inline composer.json]: I'm not sure about this 🤔 
We can do it, but then is it really a fork ? Or rather an independent project ?

It's going to change what we require in the composer.jsob

Comment (PierreRambaud) [reply composer.json]: We're also able to keep the same name and change the vcs path in our composer.json

Comment (matks) [reply composer.json]: @eternoendless Should we go for a prestashop/guzzle-bundle or do we stick to csa/guzzle-bundle and use the `vcs` to load our own ?

Comment (eternoendless) [reply composer.json]: I think we should keep the original name and use VCS. As [stated by Packagist](https://packagist.org/packages/submit):

> Do not submit forks of existing packages. If you need to test changes to a package that you forked to patch, use VCS Repositories instead. If however it is a real long-term fork you intend on maintaining feel free to submit it.

This fork exists only to provide continuous support for Guzzle 5 in PrestaShop 1.7. Once we start PrestaShop 8, we'll be able to upgrade Guzzle and drop this bundle.

**Comment by matks** [comment]
👍 

Then we can close this PR and create a new 1.3 release so we can target it in PrestaShop Core composer.json

**Pull request closed without merging**